app/models/activity.rb

```diff
@@ -455,14 +451,6 @@ def cache_roda_identifier!
     end
   end
 
-  def roda_identifier_compound=(roda_identifier)
```

Comment:
Good call, I think it's worth keeping in in that case

Reply:
How much do we want to enforce the read-only-ness of the automagically RODA identifier? If I read the tests right, users cannot edit it via the UI, is that right? Do we want to reserve the right to be able to edit it via console?

Reply:
I did put this line back in, but it borked a lot of tests. As you say, there is no longer any way to edit the RODA Identifier via the UI now, so I think it makes sense to not worry about guarding against it being changed, otherwise we'd have to add guards for a whole heap of other things we consider immutable.